models.py

```diff
@@ -20,7 +26,7 @@ class Story(Base):
     __tablename__ = 'stories'
     id = Column(Integer, primary_key=True)
     story = Column(String)
```

Comment:
Any reason for using the story field here, is it an attribute of user?

Reply:
The story field was there in the initial commit. Removing it would cause an error in the user model as it already has a relationship the story defined.